Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeSwitcher.cpp

```diff
@@ -246,23 +246,28 @@ namespace AzToolsFramework::ComponentModeFramework
         {
             componentData = componentDataIt;
         }
+     
+        if (buttonId == m_transformButtonId)
```

Comment:
suggestion: A guide comment here to indicate what transform mode is special would definitely be helpful for future maintainers

Reply:
way better now thanks! 🙂